Packs/Vectra_AI/Integrations/VectraDetect/VectraDetect.py

```diff
@@ -737,9 +737,17 @@ def extract_detection_data(detection: Dict[str, Any]) -> Dict[str, Any]:
     source_host = detection.get('src_host')
     source_host_id = source_host.get('id') if source_host else None
 
+    summary = detection.get('summary')
+    if summary:
+        description = summary.get('description', "")
+        dst_ips = summary.get('dst_ips', "")
+        dst_ports = summary.get('dst_ports', "")
+
```

Comment:
```suggestion
    summary = detection.get('summary')
    data_dict = {}
    if summary:
        if detection.get('category'):
            data_dict['Category'] = detection.get('category')
        if summary.get('description', ""):
            data_dict['description'] = summary.get('description', "")
        if summary.get('dst_ips', ""):
            data_dict['dst_ips'] = summary.get('dst_ips', "")
        if summary.get('dst_ports', ""):
            data_dict['dst_ports'] = summary.get('dst_ports', "")
```
Taking back what I said, sorry!

Reply:
Hi @YuvHayun , if you're OK I've done that, it's quicker and allow me to see all value into a single dict (with ordered keys) and won't change the others parts. Because `summary` key is always present, so it's mainly for PyPy to be happy.
```
summary = detection.get('summary')
if summary:
    description = summary.get('description')
    dst_ips = summary.get('dst_ips')
    dst_ports = summary.get('dst_ports')
else:
    description = dst_ips = dst_ports = None
```

Reply:
Thx for your `remove_empty_elements` trick on your committed fix @YuvHayun.
I will use that one more often :)